docs/topics/patterns/token_id_webhook.md

```diff
@@ -0,0 +1,361 @@
+        );
+      }
+    } else if (eventsPublishToken instanceof GenerateDisposableToken.Error) {
+      throw new Error(
+        `An error occurred while attempting to call generateApiKey with disposable token scope: ${eventsPublishToken.errorCode()}: ${eventsPublishToken.toString()}`
+      );
+    }
+    return {
+      statusCode: 200,
+      headers: {
+        'Content-Type': 'application/json',
+        'Access-Control-Allow-Origin': '*',
+      },
+      body: '{}',
+    };
+  } catch (err) {
+    console.log(err);
+    return {
+      statusCode: 500,
+      body: JSON.stringify({
+        message: 'An error occurred!',
+      }),
+    };
+  }
+}
+
+async function getAuthClient(): Promise<AuthClient> {
+  const apiKeySecretName = process.env.MOMENTO_API_KEY_SECRET_NAME;
+  if (apiKeySecretName === undefined) {
+    throw new Error("Missing required env var 'MOMENTO_API_KEY_SECRET_NAME");
+  }
+  if (_authClient === undefined) {
+    const momentoApiKey = await getSecret(apiKeySecretName);
+    console.log('Retrieved secret!');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+    _authClient = new AuthClient({
+      credentialProvider: CredentialProvider.fromString({apiKey: momentoApiKey}),
+    });
+  }
+  return _authClient;
+}
+
+async function getTopicClient(disposableTokenKey: string): Promise<TopicClient> {
+  if (_topicClient === undefined) {
+    console.log('Retrieved secret!');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+    _topicClient = new TopicClient({
+      configuration: Configurations.Lambda.latest(),
+      credentialProvider: CredentialProvider.fromString({apiKey: disposableTokenKey}),
+    });
+  }
+
+  return _topicClient;
+}
+
+async function getSecret(secretName: string): Promise<string> {
+  if (!_cachedSecrets.has(secretName)) {
+    const secretResponse = await _secretsClient.send(new GetSecretValueCommand({SecretId: secretName}));
+    if (secretResponse) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      _cachedSecrets.set(secretName, secretResponse.SecretString!);
+    } else {
+      throw new Error(`Unable to retrieve secret: ${secretName}`);
+    }
+  }
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  return _cachedSecrets.get(secretName)!;
+}
+
+```
+### Conclusion
+
+By integrating Momento Topics, webhooks, and disposable tokens, you can create secure and stateless asynchronous systems. This convention can be applied to a variety of use-cases such as:
+
+- Multi-language [chat application](https://www.gomomento.com/blog/how-to-use-webhooks-and-momento-topics-to-build-a-multi-language-chat-app)
+- Live commenting system
+- Online polls
+
+`token-id` can be used to securely identify users sending messages in all the above use-cases.
```

Comment:
This makes me think we should make `token-id` formatted as code throughout the entire page

Reply:
I did a `grep` and can confirm it's formatted as such